code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -153,6 +152,10 @@
 	else
 		. += "<span class='notice'>It is spent.</span>"
 
+/obj/item/reagent_containers/hypospray/autoinjector/epinephrine
+	list_reagents = list("epinephrine" = 10)
+	container_type = null // Only epi autoinjectors can't be drained
```

Comment:
```suggestion
	container_type = NONE // Only epi autoinjectors can't be drained
```
Preferred; even though `null` shouldn't break anything, most other containers use `NONE` so it's good for consistency